third_party/sancov.py

```diff
@@ -245,13 +245,11 @@ def PrintMissing(binary):
     if not file_list:
         Usage()
 
-    if sys.argv[1] == "print":
-        PrintFiles(file_list)
-    elif sys.argv[1] == "merge":
+    if sys.argv[1] == "merge":
```

Comment:
Does the message at start of file needs changing. I thought we use PrintFiles function.

Reply:
Done.